apps/server/src/modules/room/api/dto/response/room-list.response.ts

```diff
@@ -0,0 +1,13 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { PaginationResponse } from '@shared/controller';
+import { RoomResponse } from './room.response';
+
+export class RoomListResponse extends PaginationResponse<RoomResponse[]> {
+	constructor(data: RoomResponse[], total: number, skip?: number, limit?: number) {
+		super(total, skip, limit);
+		this.data = data;
+	}
+
+	@ApiProperty({ type: [RoomResponse] })
+	data: RoomResponse[];
```

Comment:
I would like for `data` to be more precise of what it actually is, even if its something like `roomData`